x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -147,7 +147,7 @@ export class ObservabilityAIAssistantClient {
 
         let numFunctionsCalled: number = 0;
 
-        const MAX_FUNCTION_CALLS = 3;
+        const MAX_FUNCTION_CALLS = 5;
```

Comment:
Wondering if perhaps we should make this a Kibana config? 
Or is it perhaps worse to go up to 10? No limit is bad, but how confident are we that the machine is "failing" if it makes it up to 5, compared to 10?

Specially if we add new functions that may play well together, this might be hard to catch. Unless we have some telemetry about this happening?

Reply:
10 is a lot. I hesitate to add more config. Let's see if we get reports about this.